tests/lib/formatters/table.js

```diff
@@ -0,0 +1,346 @@
+"use strict";
+
```

Comment:
Same here, please follow our conventions for comments. See other tests for examples.